src/Interfaces/IFeeding.cs

```diff
@@ -0,0 +1,11 @@
+namespace Trestlebridge.Interfaces
+{
+    //An interface for all animals that eat animal feed
+    public interface IFeeding
```

Comment:
For example: yes, you will eventually need interfaces for animals that produce compost, etc, in order to build the processing stretch goal. But where does this one come into play? Have you planned out the processing stretch goal completely, can you point to when and where this interface is necessary? If not, don't merge it. Excess interfaces will just make your code base more confusing and harder to get around.

Reply:
(you can merge it for now and delete it later if you end up not needing it, this is more for down the road)